sql/core/src/main/scala/org/apache/spark/sql/execution/metric/SQLMetrics.scala

```diff
@@ -116,8 +117,8 @@ object SQLMetrics {
     // data size total (min, med, max):
     // 100GB (100MB, 1GB, 10GB)
     val acc = new SQLMetric(SIZE_METRIC, -1)
-    acc.register(sc, name = Some(s"$name total (min, med, max (stageId (attemptId): taskId))"),
-      countFailedValues = false)
+      acc.register(sc, name = Some(s"$name total (min, med, ${attachTaskId("max", sc.conf)})"),
```

Comment:
The new indentation looks incorrect at this line?

Reply:
I also noted that, thanks.